src/test/java/uk/gov/pay/api/it/ResourcesFiltersLocalRateLimiterITest.java

```diff
@@ -36,7 +36,7 @@
 import static uk.gov.pay.api.it.fixtures.PaginatedPaymentSearchResultFixture.aPaginatedPaymentSearchResult;
 import static uk.gov.pay.api.it.fixtures.PaymentSearchResultBuilder.aSuccessfulSearchPayment;
 
-public class ResourcesFiltersITest extends PaymentResourceITestBase {
+public class ResourcesFiltersLocalRateLimiterITest extends PaymentResourceITestBase {
```

Comment:
Looks like PaymentResourceITestBase is using a real redis url, so how is this test testing local rate limiting?

Reply:
This was cause of moving RedisDockerRule to Base class. I have removed Base class dependency for ResourcesFiltersLocalRateLimiterITest and